pyiron/atomistics/structure/atoms.py

```diff
@@ -1710,6 +1741,32 @@ def plot3d(
 
         view.camera = camera
         view.background = background
+
+        if camera_axis != "x" and camera_axis != "y" and camera_axis != "z":
+            warnings.warn(
+                "Choose between x, y, or z"
+            )
+
+        zoom_out = 14  # higher = farther away
+        scale = np.eye(4) * zoom_out
+        orientation = None
+
+        if rotation_matrix is not None:
+            if np.array(rotation_matrix).shape != (4, 4):
+                raise ValueError('The shape of the rotation matrix should be (4, 4)')
+            orientation = np.dot(scale, rotation_matrix).ravel().tolist()
```

Comment:
Is the `tolist()` required? I would assume that NGLview also supports numpy arrays.

Reply:
I agree it's weird, but I'm pretty sure it is needed. NGLView internally serializes the list w/ JSON at some point, which won't work for a numpy array

Reply:
I just saw indeed it doesn't work with ndarray.